Files/MultilingualResources/Files.pt-BR.xlf

```diff
@@ -1184,12 +1191,12 @@
           <target state="translated">Ver perfis de amostra</target>
         </trans-unit>
         <trans-unit id="SettingsPreferencesAppLanguage.Header" translate="yes" xml:space="preserve">
-          <source>App language</source>
-          <target state="translated">Idioma do aplicativo</target>
+          <source>App Language</source>
```

Comment:
By the way, don't change string in `source` tag, the value is taken from Strings/en-US/Resources.resw file. Just change translation in `target` tag.